### DISCUSSION TITLE
Is there a maximum number of instances that can be run in parallel on sorry cypress for smooth execution?

### DISCUSSION THREAD

**Discussion by vpkhachane** [sorry-cypress/sorry-cypress#744] [General]
I am executing script on sorry cypress parallel. Presently running 8 instances parallel. But scripts failure rate is high. Same is working fine with 4 instance execution.
Need assistance

**Comment by agoldis** [comment]
Should be good with dozens, depending on the underlying infrastructure. 

> scripts failure rate is high.

What do you mean?

**Reply by vpkhachane** [reply]
thank you @agoldis Can you suggest for following system where currently i am running scripts
<img width="613" alt="image" src="https://user-images.githubusercontent.com/15686482/219581322-63e11af5-095c-465c-ae6e-a57cc58ad392.png">

`scripts failure rate is high`. mean many scripts getting failed but same scripts working good when run with 4 parallel instance execution

**Reply by agoldis** [reply]
what scripts? Do you mean tests?

**Reply by vpkhachane** [reply]
website test scenarios ex login, registration etc.

**Reply by agoldis** [reply]
it doesn't depend on sorry-cypress, it depends on your tests internals and the machine's configuration.,

https://docs.cypress.io/guides/getting-started/installing-cypress#Hardware

**Reply by vpkhachane** [reply]
What will be the recommended max instances for the above-mentioned system to run tests parallel successfully ?
